internal/config/mount_config.go

```diff
@@ -14,6 +14,17 @@
 
 package config
 
+import "math"
+
+const (
+	// TtlInSecsUnsetSentinel is the value internally
+	// set for metada-cache:ttl-secs
+	// when it is not set in the gcsfuse mount config file.
+	// The constant value has been chosen deliberately
+	// to be improbable for a user to explicitly set.
+	TtlInSecsUnsetSentinel int64 = math.MinInt64
```

Comment:
we can just name it TtlInSecsSentinelValue ?

Reply:
The word `sentinel` by itself doesn't mean missing/unset value. It means a special value indicating the end of a sequence ([wikipedia](https://en.wikipedia.org/wiki/Sentinel_value)), so replacing `Unset` with `Sentinel` will be misleading. If there is a golang convention in such cases, let me know.

Reply:
Wikipedia says it is end of sequence but that is too strict (it's subjective/debatale): https://stackoverflow.com/questions/56760547/what-is-the-term-for-a-sentinel-that-does-not-denote-the-end-of-a-sequence.
Why I suggested to not use Unset in the name is because sentinal often means a signal value for something. In our case, it's a signal that ttl is not set. 

However, there is no harm in using Unset in the name. So I am fine with that.